README.md

```diff
@@ -21,6 +34,49 @@ AND (alias n°2).d = "value 2"
 AND (alias n°3).a = "value 3"
 AND (alias n°3).b = "value 4"
 ```
+The request can also be implemented as a JSON query. 
+
+Similar JSON query : 
+
+```json
+{"select":{
+    "alias n°1":[
+        "a",
+        "b",
+        "c"
+    ],
+    "alias n°2":[
+        "a"
+    ]
+},
+"from":{
+    "alias n°1":"Resource type 1",
+    "alias n°2":"Resource type 2",
+    "alias n°3":"Resource type 3"
+},
+"join":{
+    "inner": {
+        "alias n°1":{
+            "d":"alias n°2"
+
+        },
+        "alias n°2":{
+            "b":"alias n°3"
+        }
+
+    }
+},
+"where":{
+    "alias n°2":{
+        "c":"value 1",
+        "d":"value 2"
+    },
+    "alias n°3":{
+        "a":"value 3",
+        "b":"value 4"
+    }
+}}
+```
```

Comment:
As discussed, in order to gain impact, I think we could:
* rework the sentence "The SQL query is written in a parser and transformed into a URL to retrieve data from a FHIR API. In the end, the JSON type data will be returned as tabular data."
* create a "Getting Started" section that says there are 2 methods to use the module: string queries and config file queries. These 2 examples could be explained in 2 sub-sections in which there would be a "functional" code (which works by copy and paste).